app/src/components/market/sections/market_list/market_home.tsx

```diff
@@ -198,6 +198,9 @@ export const MarketHome: React.FC<Props> = (props: Props) => {
   } = props
   const [state, setState] = useState<MarketStates>(MarketStates.open)
   const [category, setCategory] = useState('All')
+  const [counts, setCounts] = useState({
+    open: 0,
+  })
```

Comment:
Maybe this has another use in a future feature but as it is today it could be a single number instead of an object.

Reply:
Good catch. Yeah originally added for all of the different states. Looks like we have a closed count as well now.